torchvision/csrc/io/image/cpu/decode_jpeg.h

```diff
@@ -3,6 +3,12 @@
 #include <torch/types.h>
 #include "../image_read_mode.h"
 
-C10_EXPORT torch::Tensor decodePNG(
+namespace vision {
+namespace image {
+
+C10_EXPORT torch::Tensor decode_jpeg(
```

Comment:
Here Github fails to track the right file renames and thinks I moved png to jpeg and vice versa.